### DISCUSSION TITLE
HTTP Request spam in console

### DISCUSSION THREAD

**Discussion by Terroosh** [BlafKing/sd-civitai-browser-plus#72] [Q&A]
I'm not really sure if this is a bug or not. I was reading something recently about how deleting and recreating the venv folder can prevent issues from happening from duplicated or leftover dependencies and since I never did that before I thought I should do that. After SD recreated the venv folder I noticed one change, whenever this extension is enabled it keeps spamming console with:
```
HTTP Request: POST http://127.0.0.1:7860/api/predict "HTTP/1.1 200 OK" 
HTTP Request: POST http://127.0.0.1:7860/reset "HTTP/1.1 200 OK"
```

Clicking anything within the Civit Ai tab will output the above lines for every click. 
Clicking Apply Settings in the Settings tab will output the above lines. 
In the txt2img tab, typing in the prompt will output the above lines for every word,.
Clicking anywhere in the image preview box will output the lines.
Changing the checkpoint will output the lines x2. 
Changing clip skip, width, or height sliders will output the lines, 
Clicking the extra networks button will output the lines.

This only happens while this extension is enabled.

**Comment by BlafKing** [comment]
I'm having trouble re-creating this issue, very odd that this only happens if this extension is installed.
Since none of the functions from this script will run if you're inside other tabs.

Could you check if it still outputs the HTTP prints if you turn off Aria2 in the CivitAI Settings tab?
![2023-10-16 17_12_10](https://github.com/BlafKing/sd-civitai-browser-plus/assets/9644716/77f9ddff-4b62-43f0-908c-24107dbe5395)

**Reply by Terroosh** [reply]
It still outputs the request. There's actually not a whole lot of information on this but it doesn't look like it's an issue specifically with this extension.

https://old.reddit.com/r/StableDiffusion/comments/15o76m8/random_api_request_automatic_1111_deoldify/

This post shows the extension deoldify also outputting those lines and there were a few other comments in different reddit threads that had the output. I don't know why this appear just from recreating the venv folder. Maybe this is how it is supposed to be? It's very odd.

**Reply by BlafKing** [reply]
I would assume it has something to do with the venv folder, in case it helps, my venv folder contains a file called `pyvenv.cfg` which contains:
```
home = C:\Users\nickel\AppData\Local\Programs\Python\Python311
include-system-site-packages = false
version = 3.11.4
executable = C:\Users\nickel\AppData\Local\Programs\Python\Python311\python.exe
command = C:\Users\nickel\AppData\Local\Programs\Python\Python311\python.exe -m venv C:\Program Files\Automatic1111 SD\webui\venv
```
Don't really see how this could affect much so really odd that it's happening after deleting the venv folder

**Comment by Terroosh** [comment]
Since this happens with at least one other extension I'll close this discussion as the cause is likely with the webui and not the extension.
